PYTHON-MANIFEST.in

```diff
@@ -7,6 +7,7 @@ graft include/grpc
 graft third_party/boringssl
 graft third_party/nanopb
 graft third_party/zlib
+include src/python/grpcio/build.py
```

Comment:
I'm not a huge fan of calling this build.py, but I don't feel strongly about it.